classes/requests/helpers/class-kp-order-data.php

```diff
@@ -106,7 +106,7 @@ public function get_klarna_order_object( $iframe_options = null ) {
 			'order_amount'      => $this->order_data->get_total(),
 			'order_tax_amount'  => $this->order_data->get_total_tax(),
 			'order_lines'       => $this->get_klarna_order_lines_array(),
-			'customer'          => get_klarna_customer( $this->customer_type ),
+			'customer'          => apply_filters( 'kp_set_customer_type', get_klarna_customer( $this->customer_type ), $this->customer_type ),
```

Comment:
I would suggest putting the filter in the value that is retuned by the `get_klarna_customer` function instead, since then any other use of that function will be covered in the future as well.